src/components/keyboard/NumberButton.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Dimensions, Pressable, StyleSheet, Text } from 'react-native';
+import { useTheme } from '../../styles/themes';
+import type { KeyboardActions } from './Keyboard';
+
+const { height: SCREEN_HEIGHT } = Dimensions.get('screen');
+
+interface Props {
+    value: string;
+    onPress: (action: KeyboardActions, input?: string) => void;
+}
+
+export const NumberButton = ({ value, onPress }: Props) => {
+    const theme = useTheme();
+
+    const styles = StyleSheet.create({
+        button: {
+            height: SCREEN_HEIGHT * 0.0671,
```

Comment:
Why does a button's height has to be 6.71% of the screen height?

I think the constraint on the size of the button should be hold by the Keyboard component.

Reply:
i picked the height and divided by the screen height on figma

so i got 43/640 == 0.0671